### PR TITLE
Add Cider Plugin

### PR DESCRIPTION
**Pull request by Monochromish**
Control Cider and it's playback with Flow Launcher.

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1167,5 +1167,20 @@
...
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider",
+        "UrlDownload": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/releases/download/v1.0.0/Flow.Launcher.Plugin.Cider.zip",
+        "UrlSourceCode": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/Monochromish/Flow.Launcher.Plugin.Cider@main/Assets/app.png",
+        "Tested": true,
+        "LatestReleaseDate": "2022-10-19T12:32:10.499Z",
+        "DateAdded": "2022-10-19T12:32:10.499Z"
+    },
```
Comment (jjw24) [inline plugins.json:1185]:
```suggestion
    }
```

File: plugins.json [hunk 2]
```diff
@@ -1167,5 +1167,20 @@
...
+        "Author": "Monochromish",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider",
+        "UrlDownload": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/releases/download/v1.0.0/Flow.Launcher.Plugin.Cider.zip",
+        "UrlSourceCode": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/Monochromish/Flow.Launcher.Plugin.Cider@main/Assets/app.png",
+        "Tested": true,
+        "LatestReleaseDate": "2022-10-19T12:32:10.499Z",
+        "DateAdded": "2022-10-19T12:32:10.499Z"
```
Comment (jjw24) [inline plugins.json:1184]:
```suggestion
```

Comment (jjw24) [reply plugins.json:1184]: Hey how's it going?

These are not required, they are automatically generated once merged in.

File: plugins.json [hunk 3]
```diff
@@ -1167,5 +1167,17 @@
...
+        "ID": "4e2018ca-4ed0-11ed-bdc3-0242ac120002",
+        "Name": "Cider",
+        "Description": "Control Cider and it's playback with Flow Launcher",
+        "Author": "Monochromish",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider",
+        "UrlDownload": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/releases/download/v1.0.0/Flow.Launcher.Plugin.Cider.zip",
+        "UrlSourceCode": "https://github.com/Monochromish/Flow.Launcher.Plugin.Cider/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/Monochromish/Flow.Launcher.Plugin.Cider@main/Assets/app.png",
```
Comment (jjw24) [inline plugins.json:1181]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/Monochromish/Flow.Launcher.Plugin.Cider@main/Assets/app.png"
```

**Review by jjw24 — APPROVED** [review]

**Comment by Monochromish** [comment]
Thank you!
Also since this is my first plugin for Flow Launcher, I was wondering if I only needed to change the version in plugins.json and package.json and run workflow, or do something else in order to push an update globally.

**Comment by jjw24** [comment]
Looking at the workflow in your plugin repo, it will publish a new release when you merge your new pr to main, when doing so make sure you update the version in your plugin.json to a higher version than your current release, for example 1.0.1 and the workflow will automatically publish a new v1.0.1 release for you.

This repo will check for newer plugin version published in your GitHub Releases, if you have published a new v1.0.1,  it will be automatically updated here. So there is no need to submit anything for your plugin updates.

Hope this helps 😄

**Comment by Monochromish** [comment]
> Looking at the workflow in your plugin repo, it will publish a new release when you merge your new pr to main, when doing so make sure you update the version in your plugin.json to a higher version than your current release, for example 1.0.1 and the workflow will automatically publish a new v1.0.1 release for you.
> 
> This repo will check for newer plugin version published in your GitHub Releases, if you have published a new v1.0.1, it will be automatically updated here. So there is no need to submit anything for your plugin updates.
> 
> Hope this helps 😄

Thank you so much!

**Pull request merged**